### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#244] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152314906-6ea68a23-4ef2-4e09-bbde-95f0b6cc703b.png)

**Comment by GattoBrizzolato** [comment]
![WhatsApp Image 2022-06-07 at 22 45 23](https://user-images.githubusercontent.com/98970164/172479684-5d2c2b1c-2f4d-4386-9a9f-49927cf4b10b.jpeg)

**Comment by Elia-Belli** [comment]
**Definizione Funzione Invertibile**
Per come l'abbiamo definita noi una funzione f: X->Y è invertibile se esiste g: Y->X tale che::

- g o f è l'identità su X (per ipotesi f è iniettiva allora per la Caratterizzazione dell'Iniettività tale condizione è verificata)
- f o g è l'identità su Y (affinchè questa sia verificata f deve essere suriettiva per la Caratterizzazione della Suriettività)

Quindi per rendere valida la tesi bisogna aggiungere all'ipotesi che f sia anche suriettiva.

**Osservazione**
Se per ipotesi f è solo iniettiva e vogliamo definire una funzione g che sia la sua inversa associando ad ogni elemento del codominio Y la sua preimmagine in X, nel caso ci fossero elementi di Y non appartenenti all'immagine di X via f, allora g non sarebbe una funzione (per definizione).
